Update languages.yaml with French translation

Fazarel opened the pull request:
There's certainly room for improvement (especially when it comes to translating gallery options), but here's a suggested French translation.

I don't know if it's necessary, but I tried to frame the text with quotation marks whenever it contained an apostrophe.

pmoreno-rodriguez commented:
Thank you for your cooperation.
As soon as I can, I will integrate your translation and the rest of the changes made for the next version

pmoreno-rodriguez commented:
HI @Fazarel. 
Your French translation has some problem with the indentation of the languages.yaml file, and the translations are not showing in the Grav admin panel.

Could you check that it works correctly on some local website and then modify the Pull Request?

Fazarel commented:
I I've just modified languages.yaml, I've added the missing indentation (before ADMIN)
By the way, I noticed that a block was missing in the Spanish translation (MENU) and therefore also in the French translation, since I'd copied and pasted it. It's been added.
I tested it and it works on my installation, website and admin.
